src/showcase/app/variant-switch.ts

```diff
@@ -53,7 +53,7 @@ export class VariantSwitch implements CanActivate, OnDestroy {
         } else {
           document.documentElement.classList.add(`sbb-lean`);
         }
-        ɵvariant.next(value === 'standard' ? 'lean' : 'standard');
+        ɵvariant.next(value === 'standard' ? 'standard' : 'lean');
```

Comment:
Was this wrong before?

Reply:
Yes, it was a bug in my dark mode pull request.